lib/tasks/scripts/get_bsnl_account_balance.rb

```diff
@@ -0,0 +1,27 @@
+class GetBsnlAccountBalance
+  # (20_000 * 3 patients from current experiments + 15_000 patients from stale experiments) * 4 segments per message
+  # We're unlikely to actually hit this number on any given day, but we'd like to alert early if we're close to the limit
+  MAX_DAILY_MESSAGE_SEGMENT_COUNT = 75_000 * 4
```

Comment:
This is a bit on the higher side, with these many SMS we probably have only 5 days remaining. 75_000 * 3 is a bit more realistic (still high).

Reply:
Alright, I'll change it to `* 3`